### PR TITLE
[ISSUE-161] Release 6.1.1

### PR DESCRIPTION
**Pull request by dsupure**
#### Bug fixes and enhancements:

* [NSTK-2256] Add missing toleration to db pods
* [NSTK-2259] Support DB pods StatefulSet node-affinity and topology
* [#90](https://github.com/purestorage/pso-csi/issues/90) Enhance PSO log debug level
* [#141](https://github.com/purestorage/pso-csi/issues/141) Set Default Cockroach DB version to v20.2.6

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v6.1.1 (04/12/2021)
+
+#### Bug fixes and enhancements:
+
+* [NSTK-2256] Add missing toleration to db pods
+* [NSTK-2259] Support DB pods StatefulSet node-affinity and topology
```
Comment (haibinxie) [inline CHANGELOG.md:8]: please link to the doc

**Review by haibinxie — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v6.1.1 (04/12/2021)
+
+#### Bug fixes and enhancements:
+
+* [NSTK-2256] Add missing toleration to db pods
```
Comment (Pure-AdamuKaapan) [inline CHANGELOG.md:7]: These are internal issue numbers and don't mean anything to customers: we should create issues to represent them or just put NO-ISSUE

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+* [NSTK-2259] [Support DB pods StatefulSet node-affinity and topology](https://github.com/purestorage/pso-csi/blob/master/docs/db-topology.md)
+* [#90](https://github.com/purestorage/pso-csi/issues/90) Enhance PSO log debug level
```
Comment (Pure-AdamuKaapan) [inline CHANGELOG.md:9]: [nit] I think just putting #90 by itself will make it a link, but I'm fine leaving it as a link like this.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+* [#141](https://github.com/purestorage/pso-csi/issues/141) Set Default Cockroach DB version to v20.2.6	
+
+#### Known issues:
+* Snapshots created from imported volume don't show ready to use.
+* db volume is mounted as read-only and result in CrashLoopBackOff status of a db pod.
```
Comment (Pure-AdamuKaapan) [inline CHANGELOG.md:14]: [nit] Spell out as "database" and capitalize "D" (for the first instance)

**Review by Pure-AdamuKaapan — COMMENTED** [review]
A few textual nits on the changelog but otherwise looks good!

**Pull request merged**